src/Compilers/CSharp/Test/Semantic/Semantics/NativeIntegerTests.cs

```diff
@@ -68,6 +68,7 @@ public void LanguageVersion()
 }";
 
             var comp = CreateCompilation(source, parseOptions: TestOptions.Regular8);
+            Assert.False(comp.Assembly.RuntimeSupportsNumericIntPtr);
```

Comment:
>Assert.False [](http://example.com/codeflow?start=12&length=12)

Will this fail when tests are run with .NET 7?

Reply:
That test would pass on .NET 7, but I wanted to add some explicit checks that would fail if the tests in this file get updated to .NET 7. 
I think we'll want all the tests in this file to remain on .NET 6 (including this test).